lib/kinesis-decoder.js

```diff
@@ -72,8 +72,15 @@ exports.decode = async function(str) {
     const data = JSON.parse(json)
     const messages = (data.logEvents || []).map(e => {
       const parts = e.message.split('\t')
-      const msg = JSON.parse(parts[2])
-      return {...msg, time: e.timestamp}
+
+      // format [time, guid, json] or [time, guid, level, json]
+      if (parts.length === 3) {
+        return {...JSON.parse(parts[2]), time: e.timestamp}
```

Comment:
So to _only_ process the records with loglevel info ... just replace this line with:

```
throw new Error(`Skipping non-INFO record`)
```